packages/react/src/button/Button.css.ts

```diff
@@ -41,6 +41,7 @@ export const button = recipe({
           {
             outline: `2px solid ${theme.colors["outline.brand"]}`,
             outlineOffset: "1px",
+            zIndex: "1",
```

Comment:
let's use a theme token here
```suggestion
            zIndex: themes.zIndex["10"],
```